backstory/src/main/java/com/google/sps/servlets/data/ImagesManagerFactory.java

```diff
@@ -0,0 +1,6 @@
+import com.google.sps.images.ImagesManager;
+
+/**
+ * Factory pattern for object/mock creation.
+ */
+public interface ImagesManagerFactory { public ImagesManager newInstance(); }
```

Comment:
Additionally, all of these factories have no Javadocs for their newInstance() method.